app/assets/stylesheets/global.sass

```diff
@@ -3162,3 +3163,19 @@ nav.navbar.navbar-default ul.dropdown-menu.cycle
     .forgot-password
       font-size: 14px
       line-height: 18px
+
+.material-icon
+  font-family: "Material Icons"
+  font-weight: normal
+  font-style: normal
+  font-size: 24px
+  display: inline-block
+  line-height: 1
+  text-transform: none
+  letter-spacing: normal
+  word-wrap: normal
+  white-space: nowrap
+  direction: ltr
+  -webkit-font-smoothing: antialiased
+  text-rendering: optimizeLegibility
+  font-feature-settings: "liga"
```

Comment:
but useless lol the google fonts file already defines `.material-icons` so I could've just used it here. Fixed that on `1ed3e2a`